app/src/SVGs/Hamburger.tsx

```diff
@@ -1,22 +1,21 @@
 import React from 'react'
 
-const Hamburger: React.FC<{ width?: number }> = ({ width = 40 }) => {
+const Hamburger: React.FC<{ width?: number }> = ({ width = 20 }) => {
   return (
     <svg
       width={width}
-      viewBox='0 0 40 40'
+      height='12'
```

Comment:
Height should be omitted from the attributes since we already have width and viewbox